pom.xml

```diff
@@ -47,6 +47,12 @@
     <properties>
         <java.version>11</java.version>
 
+        <enrollment-server.version>1.5.0-SNAPSHOT</enrollment-server.version>
+        <powerauth-cmd.version>1.5.0-SNAPSHOT</powerauth-cmd.version>
```

Comment:
Waiting for publishing
- https://github.com/wultra/powerauth-cmd-tool/actions/runs/4229828597
- https://github.com/wultra/powerauth-webflow/actions/runs/4229804893